src/main/reactjs/src/configs/i18n.ts

```diff
@@ -9,6 +9,9 @@ import LanguageDetector from 'i18next-browser-languagedetector';
 import transFI from 'public/i18n/fi-FI.json';
 import transSV from 'public/i18n/sv-SE.json';
 import transEN from 'public/i18n/en-GB.json';
+import langsFI from 'public/i18n/koodisto/langs/fi-FI.json';
```

Comment:
Näitä voisi nimetä uudelleen myös toisessa taskissa niin että "koodisto" olisi mukana.